IGI.Lab2/src/CarPark.ConsoleUI/ConsoleService/MenuService/MenuConsoleService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using CarPark.ConsoleUI.Interfaces;
+
+namespace CarPark.ConsoleUI.ConsoleService.MenuService
+{
+    public class MenuConsoleService : IConsoleService
+    {
+        private readonly CarsConsoleService _carsConsoleService;
+        private readonly ContractConsoleService _contractConsoleService;
+        private readonly AccidentConsoleService _accidentConsoleService;
+
+        public MenuConsoleService(
+            CarsConsoleService carsConsoleService,
+            ContractConsoleService contractConsoleService,
+            AccidentConsoleService accidentConsoleService)
+        {
+            _carsConsoleService = carsConsoleService;
+            _contractConsoleService = contractConsoleService;
+            _accidentConsoleService = accidentConsoleService;
+        }
+
+        public void ConsoleMenu()
+        {
+            while (true)
+            {
+                try
+                {
+                    Console.Clear();
+                    ConsolePrintMenu();
+
+                    var menuItemSelection = int.Parse(Console.ReadLine() ?? throw new InvalidOperationException());
+
+                    switch (menuItemSelection)
+                    {
+                        case 1:
+                            {
```

Comment:
Do you really need brackets here?)

Reply:
Yeah, for better readability))))

Reply:
Okay, but why you have brackets in this console service only?)
It should be consistent across all console services.
PS I recommend to remove these brackets because it's useless for one line code.

Reply:
fixed

Reply:
Sorry, but I don't see commit.

Reply:
fixed